guides/manual-installation.md

```diff
@@ -587,6 +587,143 @@ docker run -d \
+
+In order to use an existing SAML IDP, some requirements must be met, configuring the IDP and starting Trento in a specific way. Follow the next instructions to properly setup both.
+
+#### SAML IDP setup
+
+Configure the existing IDP with the next minimum options to be able to connect with Trento's Service Provider (SP).
+
+##### SAML user profile
+
+Users provided by the SAML installation must have some few mandatory attributes in order to login in Trento. All of them are mandatory, even though their name is configurable.
+The user profile must include attributes for: username, email, first name and last name.
+This attributes must be mapped for all users wanting to connect to Trento.
+
+By default, Trento expects the `username`, `email`, `firstName` and `lastName` attribute names. All these 4 attribute names are configurable using the next environment variables, following the same order: `SAML_USERNAME_ATTR_NAME`, `SAML_EMAIL_ATTR_NAME`, `SAML_FIRSTNAME_ATTR_NAME` and `SAML_LASTNAME_ATTR_NAME`.
+So for example, if the IDP user profile username is defined as `attr:username`, `SAML_USERNAME_ATTR_NAME=attr:username` should be used.
+
+##### Signing keys
+
+Commonly, SAML protocol messages are signed with SSL. This is optional using Trento, and the signing is not required (even though it is recommended).
+If the IDP signs the messages, and expect signed messages back, certificates used by the SP (Trento in this case) must be provided to the IDP, the Certificate file in this case.
+
+For this reason, Trento already provides a certificates set created during the installation. When Trento is installed the first time (does not matter the installation mode) the certificates are created, and the public certificate file content is available in the `http://localhost:4000/api/public_keys` route. Copy the content of the certificate from there, and provide it to the IDP. This way, the IDP will sign and verify the messages sent by both ends.
+
+When this certificate is used, once it is provided to the IDP, the IDP recreates its own `metadata.xml` file, which defines which certificate is used to sign the messages by both sides. At this point, Trento Web must be restarted to use the new `metadata.xml` content. If `SAML_METDATA_CONTENT` option is used, this variable content must be updated with the new content. In the other hand, if `SAML_METADATA_URL` is used, the new content is automatically obtained. Otherwise, the communication fails as they don't recognize the messages signature.
+
+**This restart must be done manually, by the admin. If the installation is done by a `rpm`, restarting the `systemd` daemon. If the installation is done using `docker`, the container must be restarted.**
+
+##### SAML redirect URI
+
+Once the login is done succesfully, the IDP redirects the session back to Trento. This redirection is done to `https://trento.example.com/sso/sp/consume/saml`, so this URI must be set as valid in the IDP.
```

Comment:
Did you mean URI or URL?

Reply:
URI.
Even though they are the similar, they are not the same. In this case is we refer to URIs